packages/content-format/src/nodes/HeadingNode.ts

```diff
@@ -26,6 +33,22 @@ export namespace HeadingNode {
         return ComposedElement.isComposedElement(value, type);
     }
 
+    export function isTitleHeadingNode<Heading extends HeadingNode<Child>, Child extends Node>(
```

Comment:
Unfortunately I wasn't able to test it in the content renderer package, because linking the `@prezly/story-content-format` module worked, but it wouldn't resolve the underlying imports from `@prezly/content-format` 😕

Reply:
I've checked that the typeguards still worked, just wasn't able to compile it due to the mentioned import not being resolved.